browser/src/control/Control.ContextToolbar.ts

```diff
@@ -0,0 +1,231 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.Control.NotebookbarWriter - definition of notebookbar content in Writer
```

Comment:
comment not updated, please add note hopw to find this component in the app by user - helps later for unfamiliar devs